src/objective.jl

```diff
@@ -24,6 +24,14 @@ Return the objective value after a call to `optimize!(model)`.
 """
 objective_value(model::Model)::Float64 = MOI.get(model, MOI.ObjectiveValue())
 
+"""
+    dual_objective_value(model::Model)
+
+Return the value of the objective of the dual problem after a call to
+`optimize!(model)`.
```

Comment:
Maybe a TODO, but the docstrings for `objective_bound` and `dual_objective_value` should both mention the error cases, i.e., what happens if the attribute isn't available.